spec/models/disc_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe Disc do
+  it {should belong_to :producers}
```

Comment:
I did some figuring out of some more validation type tests and lines that could be added to each model. Let me know if you want to know abou this.